studio-docs/source/contracts.mdx

```diff
@@ -206,17 +209,21 @@ Whenever Apollo Studio attempts to create or update your contract schema, it mig
 
 | Error | Description |
 |-------|-------------|
+| `ADD_DIRECTIVE_DEFINITIONS_IF_NOT_PRESENT` | An error occurred adding directive definitions for `@tag`, `@inaccessible`, and core directive usages. |
+| `DIRECTIVE_DEFINITION_LOCATION_AUGMENTING` | An error occured augmenting the directive definition for `@tag` to support `OBJECT`, `FIELD_DEFINITION`, `INTERFACE` and `UNION`. |
 | `INPUT_VALIDATION` | The contract is attempting to include and exclude the same tag.  |
 | `PARSING` | After including and excluding fields, the resulting contract schema failed to parse. |
 | `PARSING_TAG_DIRECTIVES` | Studio encountered an error while trying to obtain all uses of `@tag` from the source variant schema. |
 | `EMPTY_OBJECT_AND_INTERFACE_MASKING` | All of an object or interface type's fields were excluded, and an error occurred while excluding the entire type.  |
 | `EMPTY_UNION_MASKING` | All of a union type's included types were excluded, and an error occurred while excluding the entire union. |
 | `PARTIAL_INTERFACE_MASKING` | An interface field's return type was excluded, and an error occurred while excluding that interface field. |
 | `SCHEMA_RETRIEVAL` | Studio encountered an error while retrieving the source variant's schema. It might not yet have a valid composed schema. |
-| `TAG_MATCHING` | Studio encountered an error while determining which tags to include and exclude. The contract might be attempting to include and exclude the same tag. |
+| `TAG_INHERITING` | Studio encountered an error while attempting to add parent tags to fields.  |
+| `TAG_MATCHING` | Studio encountered an error determining which types and fields should be inaccessible based on their tags. |
 | `TO_FILTER_SCHEMA` | Studio failed to generate and return a contract supergraph schema for an unknown reason. |
 | `TO_API_SCHEMA` | Studio encountered an error while attempting to generate an API schema from the contract variant's supergraph schema. |
 | `UNKNOWN` | An unknown error occurred. |
+| `VERSION_CHECK` | The Federation version used is not supported by contracts. |
```

Comment:
At this point the table is _almost_ alphabetically sorted. Would you mind taking it the rest of the way in this PR? 😄